src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -118,10 +122,10 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
   }
 
   /* Return value here is a tuple: (workflow name, call name) */
-  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Future[(String, String)] = {
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Try[(String, String)] = {
```

Comment:
I thought we were talking about moving this sort of logic into a proper FQN class?